apidoc/Titanium/UI/iOS/iOS.yml

```diff
@@ -1710,3 +1713,18 @@ properties:
     type: Number
     since: "8.0.0"
     permission: read-only
+
+---
+name: SystemImageParameters
+summary: |
+    Dictionary object of parameters for the 2nd parameter of the <Titanium.UI.iOS.systemButton> method
+    to pass additional configuration options like `weight` and `size`.
+properties:
+  - name: weight
+    summary: |
+        The symbol weight to use for the SF Symbol.
+        Can be one of `ultralight`, `light`, `thin`, `normal`, `semibold`, `bold`, `heavy` or `black`.
```

Comment:
I think we can make it constant. So you don't have to write utility function to convert it in constants in TiUtils.

Reply:
Agree, but since it's iOS only right now and people are used to use `fontWeight: 'bold` etc, I made it a bit more "handy" to use. Maybe both like for `textAlign`?